src/backend/cdb/cdbmutate.c

```diff
@@ -1193,7 +1194,6 @@ add_slice_to_motion(Motion *motion,
 				/* broadcast */
 				motion->plan.flow = makeFlow(FLOW_REPLICATED, numsegments);
 				motion->plan.flow->locustype = CdbLocusType_Replicated;
-
```

Comment:
restore this line

Reply:
This line should be removed because it breaks motion scaling logic

Reply:
Done